.github/workflows/ci.yml

```diff
@@ -23,3 +23,15 @@ jobs:
 
     - name: Run CI script
       run: yarn run ci
+
+    - name: Collect coverage
+      if: matrix.node == '20'  
+      run: yarn run merge-cover
+
+    - name: Coveralls
+      if: matrix.node == '20'  
```

Comment:
The CI script runs twice, on push and PR. It uploads the coverage twice. Could we do it only once, on push?

Reply:
Good catch 😉
I limited publishing only for 'push' event.